mmf/configs/models/mmf_transformer/defaults.yaml

```diff
@@ -0,0 +1,22 @@
+model_config:
```

Comment:
We should point to ConfigSchema definition here.

Reply:
Added to base transformer class